2019-governance-committee-candidates.md

```diff
@@ -0,0 +1,20 @@
+# 2019 OpenTelemetry Governance Committee Candidates
+
+In alphabetical order:
```

Comment:
can you add a link for ratification form?

Reply:
Good idea! Added at the bottom.